acq4/devices/AxoPatch200/AxoPatch200.py

```diff
@@ -561,13 +594,24 @@ def getChanHolding(self, chan):
 
         
 class AxoPatchDevGui(QtGui.QWidget):
-    def __init__(self, dev):
+    def __init__(self, dev,AxoPatchVersion):
         QtGui.QWidget.__init__(self)
         self.dev = dev
         self.ui = Ui_devGui()
         self.ui.setupUi(self)
         self.ui.vcHoldingSpin.setOpts(step=1, minStep=1e-3, dec=True, suffix='V', siPrefix=True)
         self.ui.icHoldingSpin.setOpts(step=1, minStep=1e-12, dec=True, suffix='A', siPrefix=True)
+        if AxoPatchVersion == '200A':
+            self.ui.modeCombo.setItemText(0, _translate("devGui", "V-Clamp", None))
```

Comment:
You can replace these with `self.ui.modeCombo.setItemText(0, "V-Clamp")`, which will allow you to remove the `_translate` definition above (we're only operating in one language anyway).